config/maintainers.json

```diff
@@ -1,64 +1,64 @@
 {
   "maintainers": [
     {
-      "github_username": "kotp",
-      "show_on_website": false,
```

Comment:
These two are the most important and should remain in this order at the beginning.